smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSymbolProvider.kt

```diff
@@ -118,22 +116,16 @@ class SwiftSymbolProvider(private val model: Model, val swiftSettings: SwiftSett
 
     override fun shortShape(shape: ShortShape): Symbol = numberShape(shape, "Int16", "0")
 
-    override fun bigIntegerShape(shape: BigIntegerShape): Symbol = createBigSymbol(shape, "[UInt8]")
-
-    override fun bigDecimalShape(shape: BigDecimalShape): Symbol = createBigSymbol(shape, "Complex<Double>")
+    override fun bigIntegerShape(shape: BigIntegerShape): Symbol = numberShape(shape, "Int", defaultValue = "0")
 
-    private fun createBigSymbol(shape: Shape?, symbolName: String): Symbol {
-        return createSymbolBuilder(shape, symbolName, namespace = "ComplexModule", boxed = true)
-            .addDependency(SwiftDependency.BIG)
-            .build()
-    }
+    override fun bigDecimalShape(shape: BigDecimalShape): Symbol = numberShape(shape, "Double", "0.0")
```

Comment:
BigInteger and BigDecimal changed to `Int` and `Double`, as previously mentioned